packages/fiori-elements-writer/src/translations/fiori-elements-writer.i18n.json

```
@@ -9,6 +9,7 @@
         "unsupportedOdataVersion": "OData Version of the specified service: {{ serviceVersion }}, is not supported by the template type: {{ templateType }}",
         "unsupportedUI5Version": "Specified UI5 property '{{ versionProperty }}': {{ ui5Version }}, is not supported by the template type: {{ templateType }}. Please specify {{minRequiredUI5Version}} or above.",
         "invalidUI5Version": "Specified UI5 property '{{ versionProperty }}': {{ ui5Version }}, is not a valid semantic version",
-        "missingRequiredProperty": "A property required for application generation is not specified: '{{ propertyName }}'."
+        "missingRequiredProperty": "A property required for application generation is not specified: '{{ propertyName }}'.",
+        "errorGeneratingDefaultAnnotations": "Error generating default annotations"
```

Can you please include the error message fto help with support:
```suggestion
        "errorGeneratingDefaultAnnotations": "Error generating default annotations: {-error}"
```

---

Sorry I see you have done